packages/react/src/store/sagas/backendSaga.ts

```diff
@@ -92,22 +92,24 @@ export function* watchApiAnalysisUpdate() {
 
 function* handleInit(action: ReturnType<typeof init>) {
   yield put(ping());
-  const haveInternet:boolean = yield window.backend.checkInternetAccess();
+  const haveInternet: boolean = yield window.backend.checkInternetAccess();
   const storedSolutions = window.electron.getState("solutions", {});
   const targetFramework = getTargetFramework();
   if (!haveInternet) {
-    yield put(pushCurrentMessageUpdate({
-      messageId: uuid(),
-      groupId: "internetAccessFailed",
-      type: "error",
-      header: "No Internet Access",
-      content: "Ensure you have internet access before using the application",
-      buttonText: "View prerequisites",
-      onButtonClick: () => window.electron.openExternalUrl(externalUrls.prereq)
-    }));
+    yield put(
+      pushCurrentMessageUpdate({
+        messageId: uuid(),
+        groupId: "internetAccessFailed",
+        type: "error",
+        header: "No Internet Access",
+        content: "Ensure you have internet access before using the application",
```

Comment:
Change to: "Please make sure you have internet access before using the application". For better results, I would get input from doc writer on all customer facing messages